docs/install/pubsub-service-account.md

```diff
@@ -47,25 +47,40 @@ Service Account.
           --member=serviceAccount:cre-pubsub@$PROJECT_ID.iam.gserviceaccount.com \
           --role roles/pubsub.editor
         ```
-
-    1.  Download a new JSON private key for that Service Account. **Be sure not
+1. Access Google Cloud service. There are two methods provided to access 
```

Comment:
Configure the authentication mechanism used for accessing the Google Cloud services. Currently, we support two methods: